tests/foreman/ui/test_computeresource_rhev.py

```diff
@@ -718,17 +730,22 @@ def test_positive_rhev_vm_power_on(self):
             ['Password', self.rhev_password, 'field'],
             ['Datacenter', self.rhev_datacenter, 'special select'],
         ]
-        name = gen_string('alpha')
+        cr_name = gen_string('alpha')
         with Session(self.browser) as session:
             make_resource(
                 session,
-                name=name,
+                name=cr_name,
                 provider_type=FOREMAN_PROVIDERS['rhev'],
                 parameter_list=parameter_list
             )
+            if self.compute_resource.power_on_status(
```

Comment:
Doesn't sound logical to me. You created new compute resource, so it should be in known state. If you have not static and expected state you should raise an exception, not changing it to suitable one.